### ISSUE TITLE
--watch doesnt work on mocha 5.1.0

### ISSUE THREAD

**Issue by CoericK** [mochajs/mocha#3325]
I got these scripts
```
{
    "test": "mocha --require babel-core/register src/**/*.test.js",
    "test:watch": "yarn test --watch"
}
```

and they work perfect on mocha 5.0.5, but i just created a new project and installed mocha 5.1.0 and the `npm run test:watch` command doesn't work. I mean it doesnt give any error, it executes the tests and finish the process, if i have to run the tests again i have to run the command again and again.

Workaround: Going back to 5.0.5 for now.

**Comment by mwangaben** [comment]
I am facing the same issue with mocha 5.1.0

**Comment by boneskull** [comment]
Ugh, sorry--c5802948 got in when it shouldn't have.  I'm experimenting with some new branching and apparently got confused.

**Comment by boneskull** [comment]
Released as [v5.1.1](https://github.com/mochajs/mocha/releases/tag/v5.1.1)

**Comment by CoericK** [comment]
Thanks @boneskull !!
